src/Faker/Provider/nl_NL/Person.php

```diff
@@ -314,4 +314,33 @@ public static function prefix()
     {
         return static::randomElement(static::$prefix);
     }
+    
+    /**
+     * 11 proof idNumber
+     * @return string
+     */
+    public function idNumber()
+    {
+        $return = '';
+        $nr = [];
+        $nr[] = 0;
+        while(count($nr) < 8) {
+            $nr[] = rand(0,9);
```

Comment:
never use `rand()` in Faker, use `mt_rand()` instead. And if you want to generate a random digit, use `randomDigit` instead. Same on line 330